.evergreen/config.yml

```diff
@@ -1096,6 +1111,28 @@ functions:
           PKG_CONFIG_PATH=$PKG_CONFIG_PATH \
           LD_LIBRARY_PATH=$LD_LIBRARY_PATH
 
+  run-azure-imds-test:
```

Comment:
I expect the Azure mock server tests do not need to run in a separate Evergreen task.

Consider combining this into the tests run in run-tests.sh:

- Add the commands from `start-kms-mock-azure-imds-server` to `start-cse-servers`. `start-cse-servers` is run before every task as part of the `pre` tasks.
- Remove the `run-azure-imds-test` task.
- Add `export AZURE_MOCK_SERVERS_RUNNING=true` to the `run-tests.sh` script.
- Remove the `evg-test-azure` Makefile target.

Reply:
> Add the commands from start-kms-mock-azure-imds-server to start-cse-servers. start-cse-servers is run before every task as part of the pre tasks.

a428b6a39607290b2dd4389a7c363e79830088a8 adds the `start-kms-mock-azure-imds-server` back. Does adding to `start-cse-servers` not work as expected?